subsys/bluetooth/host/classic/sdp.c

```diff
@@ -2467,6 +2467,16 @@ static inline ssize_t sdp_get_seq_len(const uint8_t *data, size_t len)
 		return 3 + sys_get_be16(pnext);
 	case BT_SDP_SEQ32:
 	case BT_SDP_ALT32:
+		/* validate len for pnext safe use to read 32bit value */
+		if (len < 5) {
+			break;
+		}
+
+		if (len < (5 + sys_get_be32(pnext))) {
```

Comment:
Where is the magic number 5 coming from? Same for other magic numbers here, like the `3` on line 2467. Try to use `sizeof(some struct)` or meaningful preprocessor defines instead.

Reply:
> Where is the magic number 5 coming from? Same for other magic numbers here, like the `3` on line 2467. Try to use `sizeof(some struct)` or meaningful preprocessor defines instead.

BT_SDP_ALT32 mean 5 bytes for record element header.  here is the minimal changes to keep consistent

I could do more changes for better code abstraction for sdp_get_str_len and sdp_get_uuid_len etc in the future

Reply:
> With that error, my guess would be that you didn't try to push to your fork, rather you tried to push directly to the upstream tree, which is obviously forbidden.

ci just complain " UC2 Signed-off-by: must have a full name"

and I fixed that  name error, please check it again? @jhedberg